pallets/core/src/modules/trust_registry/mod.rs

```diff
@@ -288,9 +310,13 @@ pub mod pallet {
         ) -> DispatchResult {
             ensure_signed(origin)?;
 
+            let f = ActionWithNonceWrapper::wrap_fn_with_modify_removable_action(
+                Self::init_or_update_trust_registry_,
+            );
+
             init_or_update_trust_registry
                 .signed_with_signer_target(signature)?
-                .execute(ActionWrapper::wrap_fn(Self::init_or_update_trust_registry_))
+                .execute(f)
         }
 
         /// Sets the schema metadata entry (entries) with the supplied identifier(s).
```

Comment:
Would `set_schemas_metadata` update `TrustRegistriesParticipants`? Or before calling `set_schemas_metadata` with any issuer/verifier, they have to be added in `TrustRegistriesParticipants`

Reply:
Before calling `set_schemas_metadata` `Convener` has to invite all new participants to the registry using `change_participants` method.

Reply:
Can you please point out where the check is made?

Reply:
https://github.com/docknetwork/dock-substrate/blob/bba34abf089b36d37eb134087a55dd4620e85960/pallets/core/src/modules/trust_registry/update.rs#L471